packages/amazon-sumerian-hosts-babylon/src/Babylon.js/index.js

```diff
@@ -25,59 +25,59 @@ import anim from './animpack';
 
 export default {
   /**
-   * @see env
-   */
-  env,
-  /**
-   * @see Utils
+   * @see module:hosts/core.Utils
    */
   Utils,
   /**
-   * @see MathUtils
+   * @see module:hosts/core.MathUtils
    */
   MathUtils,
   /**
-   * @see Deferred
+   * @see module:hosts/core.Deferred
    */
   Deferred,
   /**
-   * @see Babylon.js/Messenger
-   */
-  Messenger,
-  /**
-   * @see Babylon.js/HostObject
-   */
-  HostObject,
-  /**
-   * @see @amazon-sumerian-hosts/core/LipsyncFeature
+   * @see module:hosts/core.LipsyncFeature
    */
   LipsyncFeature,
   /**
-   * @see GestureFeature
+   * @see module:hosts/core.GestureFeature
    */
   GestureFeature,
   /**
-   * @see Babylon.js/PointOfInterestFeature
+   * @see babylonjs/PointOfInterestFeature
```

Comment:
To keep consistency, I used lower case here. I am open for either upper or lower letter but we must remove dot signal to avoid jsdoc's error.

Reply:
Sorry, the highlighting didn't work. You've moved all "babylonjs/*" exports below "module:host/core" with the exception of PointOfInterestFeature, which remains in middle of two "module:host/core" properties. This seems like it was intended to be in the lower section around line 68 instead of here.

Reply:
ok. I got you. Moved it to the line 68 to keep consistency.